src/web/src/pages/signup.tsx

```diff
@@ -153,6 +153,14 @@ const useStyles = makeStyles((theme: Theme) =>
   })
 );
 
+enum SIGN_UP_STEPS {
+  OVERVIEW = 0,
+  BASIC_INFO = 1,
+  GITHUB_ACCOUNT = 2,
+  RSS_FEEDS = 3,
+  REVIEW = 4,
+}
+
 const SignUpPage = () => {
   const classes = useStyles();
   const [activeStep, setActiveStep] = useState(0);
```

Comment:
Now we use `enum`, this could be replaced with `useState(steps.overview)`?

Reply:
Nice catch!